MetaMask/metamask-mobile
# Fix ENS name displayed on confirm send page

Fixes: #6330

Show ENS name of to address in confirmation page.

QA Steps:
- start a send transaction on mainnet
- enter an ens address, go to next page
- enter amount, go to confirm page
- ***ENS name*** of to address should be visible
